packages/rich-list-core/src/RichListCore.ts

```diff
@@ -0,0 +1,179 @@
+import { ApiClient, BigNumber, blockchain as defid } from '@defichain/jellyfish-api-core'
+import { QueueClient } from './lib/Queue'
+import { PersistentStorage } from './lib/PersistentStorage'
+import { AddressParser } from './controller/AddressParser'
+import { NetworkName } from '@defichain/jellyfish-network'
+import { RichListItem } from '@defichain/rich-list-api-client'
+import { AccountAmount } from 'packages/jellyfish-api-core/src/category/account'
+import { ActiveAddressAccountAmount } from './controller/AddressParser/ActiveAddressAccountAmount'
+import { PersistentLinkedList } from './lib/PersistentLinkedList'
+
+export class RichListCore {
+  CATCHING_UP = false
+  RICH_LIST_LENGTH = 1000
```

Comment:
Quick question on variable naming here, usually all uppercase variables usually refers to a constant unchangeable assignment. Usually when I see variables in this format I think constants, thoughts?

Reply:
It started to be constant and I made it configurable later. Agree it should be in camel case.